graphql2/graphqlapp/contactmethod.go

```diff
@@ -72,6 +73,14 @@ func (q *Query) UserContactMethod(ctx context.Context, id string) (*contactmetho
 
 func (m *Mutation) CreateUserContactMethod(ctx context.Context, input graphql2.CreateUserContactMethodInput) (*contactmethod.ContactMethod, error) {
 	var cm *contactmethod.ContactMethod
+
+	cfg := config.FromContext(ctx)
+
+	// checks webhook url against allow list set in config
+	if input.Type == "WEBHOOK" && !cfg.ValidWebhookURL(input.Value) {
+		return cm, validation.NewFieldError("value", "Webhook URL not included in allow list.")
```

Comment:
Maybe something like "URL disallowed by administrator", an end-user will most likely not know about the "allow list".